docs/team/vn-hao.md

```diff
@@ -5,25 +5,29 @@
 
 # Project: CampusConnect
 
-to be added soon
+**CampusConnect** is a desktop application built for **NUS students** living on campus to help them **stay organised**, **stay connected**, and **make the on-campus experience at NUS stress-free**. It is **optimised for use** via **Command Line Interface (CLI)** while benefiting from a **Graphical User Interface (GUI)**.
 
 # Summary of Contributions
 
 ## Code Contributed
 
-to be added soon
+- Implement `updatephoto` command.
+  - Usage: Allow user to update photo of given contacts so that he/she can visually remember them.
+  - Here is the link to my code: [vn_hao](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=vn-hao&breakdown=false&sort=groupTitle%20dsc&sortWithin=title&since=2023-09-22&timeframe=commit&mergegroup=&groupSelect=groupByRepos&tabOpen=true&tabType=authorship&tabAuthor=VN-Hao&tabRepo=AY2324S1-CS2103T-T13-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
 ## Enhancements Implemented
 
 to be added soon
 
 ## Contributions to the User Guide
 
-to be added soon
+- Add description for `updatephoto` command in User Guide.
 
 ## Contributions to the Developer Guide
 
-to be added soon
+- Add description for `updatephoto` command in Developer Guide.
+  - Explain implemetation and logic flow of the command.
+  - Draw UML diagram to give a visual view of the logic flow.
 
 ## Contributions to Team-Based Tasks
 
```

Comment:
I think all these subsections which are just `to be added soon` can/should be removed for cleanliness